ass/build.gradle

```diff
@@ -23,7 +31,7 @@ dependencies {
     implementation "com.android.support.constraint:constraint-layout:1.1.3"
 
     implementation "com.squareup.retrofit2:retrofit:2.5.0"
-    implementation "com.squareup.retrofit2:converter-moshi:2.5.0"
+    implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
```

Comment:
why change of moshi in favor of gson?

Reply:
There were many errors in current gradle configuration with moshi + proguard rules. This is the easiest way.

Reply:
There are many of our apps that have this combination and I dont recall any problems. I would like to solve these problems instead of blindfold swap one serializer for another .

Reply:
Used moshi